apps/connect/src/App.tsx

```diff
@@ -35,6 +35,9 @@ export default function Root() {
   const messages = Object.values(messageConfig);
   return (
     <>
+      {versions.map(({ appName, version }, idx) => (
```

Comment:
if we are going to remove this, please log it at the console or store at some place, to be sure we have the info when we research bugs